semcore/dropdown-menu/__tests__/dropdown-menu.browser-test.tsx

```diff
@@ -168,3 +167,106 @@ test.describe('Dropdown-menu - On Visible controlled', () => {
     await expect(Item2).toBeFocused();
   });
 });
+
+test.describe('Dropdown-menu - Selectable radio items', () => {
+  test('Keyboard interaction', async ({ page }) => {
+    const standPath = 'stories/components/dropdown-menu/docs/examples/selectable_radio_items.tsx';
+    const htmlContent = await e2eStandToHtml(standPath, 'en');
+    await page.setContent(htmlContent);
+
+    //1st item focused when Menu expands
+    const ddMenuTrigger = await page.locator('[data-ui-name="DropdownMenu.Trigger"]');
+    await page.keyboard.press('Tab');
+    await expect(ddMenuTrigger).toBeFocused();
+    await page.keyboard.press('Enter');
+    await page.waitForTimeout(500);
+    await expect(ddMenuTrigger).not.toBeFocused();
+    const Item1 = page
+      .locator('[data-ui-name="DropdownMenu.Item.Content"]')
+      .filter({ hasText: /^Menu item 1$/ });
+    await expect(Item1).toBeFocused();
+
+    //The DD closed by Enter (no focus on intercative element)
+    await page.keyboard.press('Enter');
+    await page.waitForTimeout(500);
```

Comment:
@Valeria-Zimnitskaya What do you think about replacing timeouts with actual thing we are waiting here?

Or this is the case we can't use anything esle?

Reply:
I have replaced unnecessary timeouts in some parts of the code with actual elements or states we are waiting for. In other places, I have removed timeouts entirely as they were not needed. However, in certain cases, I left timeouts because it is a simpler solution for now. If these timeouts end up causing performance issues, i will revisit and replace them later.